.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches: [master]
```

Comment:
Should we check on all branches?

Reply:
If you check on all branches I think that it will run twice when you add commits to a PR.
See https://github.com/dandavison/delta/pull/467#issuecomment-751349615

Reply:
Just want to make sure I understand this. This would trigger when a pull request, and direct push to master. I am wonder is there anyway we can run this direct push to and PR merging to vspacecode-next, which-key-next  and master. I am asking because it would be nice to catch the lint issue when people submitting PR or us changing it directly.

Reply:
You are right, I will specify also those two branches

Reply:
is it better know? :)

Reply:
Let's try it :)